spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -361,6 +389,10 @@ private static class PositionDeltaWriteFactory implements DeltaWriterFactory {
     @Override
     public DeltaWriter<InternalRow> createWriter(int partitionId, long taskId) {
       Table table = tableBroadcast.value();
+      Map<String, DeleteFileSet> rewritableDeletes = Maps.newHashMap();
```

Comment:
What about a helper method and using it directly in statements below?

```
private Map<String, DeleteFileSet> rewritableDeletes() {
  return rewritableDeletesBroadcast != null ? rewritableDeletesBroadcast.getValue() : null;
}
```

Reply:
Good suggestion, done! Note, since it's possible now that rewritable deletes is null in the writer, I explicitly pass in the previous loader to be a function path -> null. Previously I was passing in an empty map so the lookup would return null, but i think your suggestion is better because we can avoid having to do a lookup in the map entirely